src/main/java/io/vertx/httpproxy/impl/HttpProxyImpl.java

```diff
@@ -69,6 +63,14 @@ public void handle(HttpServerRequest outboundRequest) {
     });
   }
 
+  private Future<WebSocket> resolveOriginWS(HttpServerRequest outboundRequest) {
```

Comment:
this should be renamed as it does not resolve the original WS but instead resolve the origin and then connect to the resolved address

Reply:
Related code has been fixed.